engine/schema/src/main/resources/META-INF/db/schema-42000to42010.sql

```diff
@@ -69,3 +69,8 @@ CALL `cloud`.`IDEMPOTENT_UPDATE_API_PERMISSION`('Read-Only Admin - Default', 'va
 
 CALL `cloud`.`IDEMPOTENT_UPDATE_API_PERMISSION`('Support Admin - Default', 'setupUserTwoFactorAuthentication', 'ALLOW');
 CALL `cloud`.`IDEMPOTENT_UPDATE_API_PERMISSION`('Support Admin - Default', 'validateUserTwoFactorAuthenticationCode', 'ALLOW');
+
+-- Fix dynamic setting for the network.gc.interval
+
+UPDATE `cloud`.`configuration` SET `is_dynamic`=0 WHERE  `name`='network.gc.interval';
+
```

Comment:
@andrijapanicsb Update it here (change true -> false in L4871 for isDynamic param), this'll update DB as well. https://github.com/apache/cloudstack/blob/576b97ba2215dec9a7f56201bd3912f8e19e18d3/engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java#L4870-L4871

Reply:
+1, no need for this SQL change